controller/token.go

```diff
@@ -126,6 +126,26 @@ func (c *TokenController) Generate(ctx *app.GenerateTokenContext) error {
 			ProviderType:          account.KeycloakIDP,
 			RegistrationCompleted: true,
 		}
+
+		err = transaction.Transactional(c.app, func(tr transaction.TransactionalResources) error {
+			// Using the old-fashioned service
+			err := tr.Identities().Save(ctx, &devIdentity)
```

Comment:
You could add tests to verify that the user and identity records are created, and that they are correctly linked (eg: `Identity.UserID` == generated `User.ID`)

Reply:
sure let me try to add that